client/src/components/Footer/showName.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./modal.css";
+import { ReactComponent as Github } from "./img/github-brands.svg";
+import { ReactComponent as Twitter } from "./img/twitter-brands.svg";
+import { ReactComponent as Mail } from "./img/envelope-solid.svg";
+
+function ShowName({ prop }) {
+  const [userInfo, setUserInfo] = useState({});
+
+  useEffect(() => {
+    getGitHubUserWithFetch();
+  });
```

Comment:
fix this infinite call is happening!
pass an empty array to the useEffect hook

Reply:
Issue resolved , please take a look at it again.